server/utils/nunjucksSetup.ts

```diff
@@ -45,4 +46,8 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
     const array = fullName.split(' ')
     return `${array[0][0]}. ${array.reverse()[0]}`
   })
+
+  njkEnv.addFilter('formatDate', (dateToFormat, dateFormat = 'd MMMM yyyy') => {
```

Comment:
We could add a few filters for different types of dates and keep the generic one too? Makes it easier to see and update places where it's used maybe?